web/massastation/src/pages/Index/Index.tsx

```diff
@@ -91,72 +79,36 @@ function NestedIndex({
-                </div>
-              </Button>
-              <Button
-                variant="secondary"
-                preIcon={<FiCodepen />}
-                customClass="w-96"
-                onClick={() => navigate(routeFor('store'))}
-              >
-                <div className="flex items-center mas-buttons">
-                  {Intl.t('index.buttons.explore')}
-                </div>
-              </Button>
-            </div>
-            <DashboardStation
-              theme={theme}
-              imagesDark={[
-                <Image1Dark />,
-                <Image2Dark />,
-                <Image3Dark />,
-                <Image4Dark />,
-                <Image5Dark />,
-              ]}
-              imagesLight={[
-                <Image1Light />,
-                <Image2Light />,
-                <Image3Light />,
-                <Image4Light />,
-                <Image5Light />,
-              ]}
-              components={[
-                <PluginWallet
-                  key="wallet"
-                  isActive={pluginWalletIsInstalled}
-                  status={
-                    massaPlugins?.find(
-                      (plugin: MassaPluginModel) =>
-                        plugin.name === MASSA_WALLET,
-                    )?.status
-                  }
-                  isLoading={isLoading}
-                  title="Massa Wallet"
-                  iconActive={<WalletActive />}
-                  iconInactive={<WalletInactive />}
-                  onClickActive={() =>
-                    window.open(
-                      '/plugin/massa-labs/massa-wallet/web-app/index',
-                      '_blank',
-                    )
-                  }
-                  onClickInactive={() =>
-                    urlPlugin ? handleInstallPlugin(urlPlugin) : null
-                  }
-                />,
-              ]}
-            />
+        <div className="w-fit max-w-[1760px] ">
+          <div className="flex gap-8 pb-10">
+            <Button
+              preIcon={<FiGlobe />}
+              customClass="w-96"
+              onClick={() => navigate(routeFor('search'))}
+            >
+              <div className="flex items-center mas-buttons">
+                {Intl.t('index.buttons.search')}
+              </div>
+            </Button>
+            <Button
+              variant="secondary"
+              preIcon={<FiCodepen />}
+              customClass="w-96"
+              onClick={() => navigate(routeFor('store'))}
+            >
+              <div className="flex items-center mas-buttons">
+                {Intl.t('index.buttons.explore')}
+              </div>
+            </Button>
           </div>
+          <DashboardStation
+            massaPlugins={massaPlugins}
+            pluginWalletIsInstalled={pluginWalletIsInstalled}
+            urlPlugin={urlPlugin}
+            isLoading={isLoading}
+            handleInstallPlugin={handleInstallPlugin}
+            theme={theme}
```

Comment:
it looks like you don't use theme props in DashboardStation component, maybe clean this